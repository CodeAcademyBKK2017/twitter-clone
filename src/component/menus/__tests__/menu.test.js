import Menu from '../menu.component';
import React from 'react';
import renderer from 'react-test-renderer';

describe('Menu component', () => {
  it('Menu: renders correctly', () => {
    const tree = renderer.create(<Menu />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});