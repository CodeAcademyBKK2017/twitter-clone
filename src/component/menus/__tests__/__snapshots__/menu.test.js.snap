// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Menu component Menu: renders correctly 1`] = `
<div
  className=""
>
  <h4
    className=""
  >
    
  </h4>
  <p>
    
  </p>
</div>
`;
